$NetBSD: patch-configure.ac,v 1.2 2018/09/18 14:30:17 leot Exp $

Really check for <util.h> needed by openpty(3) at least on NetBSD.

Please note that src/rlwrap.h already checked for HAVE_UTIL_H in
src/rlwrap.h but actually HAVE_UTIL_H was never defined because
util.h was not added to the AC_CHECK_HEADERS().

Also reported upstream via:

 <https://github.com/hanslub42/rlwrap/pull/85>

--- configure.ac.orig	2017-03-03 09:41:58.000000000 +0000
+++ configure.ac
@@ -102,7 +102,7 @@ AC_CHECK_PROG(STRIP,strip,strip,true)
 # Checks for header files.
 AC_HEADER_STDC
 AC_HEADER_SYS_WAIT
-AC_CHECK_HEADERS([errno.h fcntl.h libgen.h libutil.h stdlib.h string.h sched.h sys/ioctl.h sys/wait.h sys/resource.h stddef.h ])
+AC_CHECK_HEADERS([errno.h fcntl.h libgen.h libutil.h util.h stdlib.h string.h sched.h sys/ioctl.h sys/wait.h sys/resource.h stddef.h ])
 AC_CHECK_HEADERS([termios.h unistd.h stdint.h time.h getopt.h regex.h curses.h termcap.h ])
 
 AC_CHECK_HEADERS([ term.h  ncurses/term.h], , ,
